lib/ansible/cli/__init__.py

```diff
@@ -55,7 +61,7 @@ def initialize_locale():
     if not encoding or encoding.lower() not in ('utf-8', 'utf8'):
         raise SystemExit('ERROR: Ansible requires the locale encoding to be UTF-8; Detected %s.' % encoding)
 
-    fs_enc = sys.getfilesystemencoding()
+    fs_enc: str = sys.getfilesystemencoding()
```

Comment:
```suggestion
    fs_enc = sys.getfilesystemencoding()
```

You shouldn't need to add type annotations for stdlib function return values. They are already typed, and mypy can infer the types.

Reply:
TIL - thanks!